draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -148,6 +175,21 @@ SHOULD NOT be retransmitted.
 As described in (Section 3.2 of {{RFC9000}}), it MAY deliver data beyond that
 offset to the application.
 
+# Aborting reading
+
+Using the STOP_SENDING frame, RFC 9000 allows the receiver of a stream to abort
+reading and to request the sender to reset the stream. Analogously, the ENOUGH
+frame can be used to request resetting the stream at a specific offset.
+
+The receiver of an ENOUGH frame MUST reset the stream, either by sending a
+RESET_STREAM frame, or by sending a CLOSE_STREAM frame. It MAY honor the
```

Comment:
```suggestion
The receiver of an ENOUGH frame MUST reset the stream by sending a either a
RESET_STREAM frame or a CLOSE_STREAM frame, if the stream is in the "Ready" or
"Send" state. It MAY honor the
```

Reply:
Applied, but I fixed the typo.